package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gamefeeder",
-  "version": "0.13.7",
+  "version": "0.14.0",
```

Comment:
I would change this to at most a patch bump

Reply:
Fixed.